lang/snobol4/Portfile

```diff
@@ -3,30 +3,37 @@
 PortSystem          1.0
 
 name                snobol4
-version             1.2
+version             2.3.1
 revision            0
 categories          lang
 license             BSD
-maintainers         nomaintainer
+maintainers         regressive.org:snobol4
```

Comment:
can `openmaintainer` be added here so that others can perform minor changes/updates to the Portfile without your permission?